src/en/developer-leadership.md

```diff
@@ -167,7 +169,7 @@ primitives. As a charm author, this is unlikely to be the best use of your time
 -- extending the `hacluster` charm to cover your use case is likely to be more
 efficient.
 
-### ...render it likely that a long-lived process runs on one unit at a time?
+^# ...render it likely that a long-lived process runs on one unit at a time?
```

Comment:
I am changing this to ...run a long-lived process on one unit at a time.
The Note: box will do the hand waving about reliability.